src/EditorFeatures/Core.Wpf/NavigateTo/NavigateToItemProvider.Searcher.cs

```diff
@@ -229,6 +229,48 @@ private void ReportMatchResult(Project project, INavigateToSearchResult result)
                 _callback.AddItem(navigateToItem);
             }
 
+            private static string GetKind(string kind)
```

Comment:
@CyrusNajmabadi This is per our discussion in discord.
I hope I did it the correct way.